frontend/lang/en.json

```diff
@@ -21,6 +31,16 @@
         "If Persists": "If this issue persists, please report this issue by",
         "File a Bug": "filing a bug in the PACTA repository"
     },
+    "ModalFakeUsers": {
+        "Heading": "Fake Users",
+        "Subheading": "Quickly switch between profiles for testing.",
+        "Description": "Microsoft Active Directory doesn't allow us to programatically log in as other users, so you will need to manually log in as one of the users below. These are only in the Local environment (for now).",
```

Comment:
I haven't actually looked into this at all, how sure are you of this? I know the `msalInstance/PublicClientApplication` doesn't have any obvious `signInWithUserAndPassword` methods, but that doesn't mean there's nothing to be done here.